src/main/java/brandkon/product/ProductService.java

```diff
@@ -0,0 +1,95 @@
+package brandkon.product;
+
+import brandkon.brand.Brand;
+import brandkon.brand.BrandRepository;
+import brandkon.category.Category;
+import brandkon.category.CategoryRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
+    ProductRepository productRepository;
+    BrandRepository brandRepository;
+    CategoryRepository categoryRepository;
```

Comment:
```suggestion
    private final ProductRepository productRepository;
    private final BrandRepository brandRepository;
    private final CategoryRepository categoryRepository;
```
주입받는 필드에는 `private final`을 붙여 주면 더 좋습니다.